packages/zero-client/src/client/query-manager.test.ts

```diff
@@ -66,10 +66,52 @@ test('add', () => {
     },
   ]);
 
-  queryManager.add(ast, 'forever');
+  queryManager.addLegacy(ast, 'forever');
   expect(send).toBeCalledTimes(1);
 });
 
+test('add and remove a custom query', () => {
```

Comment:
We now have a lot more test coverage for addLegacy than for addCustom (covering behavior like gotCallback, recent query cache, etc). addLegacy and addCustom share logic for most of this stuff.  But in the long run addLegacy is going away. Is the thought to update these tests to use addCustom at the time we remove addLegacy?

Reply:
Let me see about sharing the same test logic for both paths.